src/sentry/models/grouptombstone.py

```diff
@@ -39,6 +39,8 @@ class GroupTombstone(Model):
         blank=True, null=True
     )
     actor_id = BoundedPositiveIntegerField(null=True)
+    times_seen = BoundedPositiveIntegerField(default=0, null=True)
+    last_seen = models.DateTimeField(default=timezone.now, null=True)
```

Comment:
If you have a default value you could make the column not-null. You might want to use `db_default` instead as that will put the default in the schema vs only in django. You can see the `DROP DEFAULT` operations in the SQL dump, and we could retain those defaults with `db_default=...`.